src/docs/PML/PMLUserGuide.adoc

```diff
@@ -1564,6 +1564,130 @@ This example trains a Mnist model, starts a service instance where the trained m
+
+AutoFeat provides a complete solution to assist data scientists to encode successfully their categorical data.
+
+In real-world problems, most of the time we require choosing one encoding method for the proper working of the model. Working with different encoders can vary the results of the model.
+
+AutoFeat currently supports the following encoding methods:
+
+- Label: converts each value in a categorical feature into an integer value between 0 and n-1, where n is the number of distinct categories of the variable.
+- Binary: stores categories as binary bitstrings.
+- OneHot: creates a new feature for each category in the Categorical Variable and replaces it with either 1 (presence of the feature) or 0 (absence of the feature). The number of the new features depends on the number of categories in the Categorical Variable.
+- Dummy: transforms the categorical variable into a set of binary variables (also known as dummy variables). The dummy encoding is a small improvement over the one-hot-encoding, such it uses n-1 features to represent n categories.
+- BaseN: encodes the categories into arrays of their base-n representation. A base of 1 is equivalent to one-hot encoding and a base of 2 is equivalent to binary encoding.
+- Target: replaces a categorical value with the mean of the target variable.
+- Hash: maps each category to an integer within a pre-determined range n_components. n_components is the number of dimensions, in other words, the number of bits to use to represent the feature. We use 8 bits by default .
+
+The most of these methods are implemented using the python link:https://contrib.scikit-learn.org/category_encoders/[Category Encoders] library.
+As we said earlier, the performance of ML algorithms depends on how categorical variables are encoded. The results produced by the model varies from different encoding techniques used. Thus, the hardest part of categorical encoding can sometimes be finding the right categorical encoding method.
+
+There are numerous research papers and studies dedicated to analyzing the performance of categorical encoding approaches to different datasets. Based on the common factors shared by the datasets using the same encoding method, we have implemented an algorithm for finding the method best suited for your data. This can help the data scientist to start encoding smarter.
```

Comment:
dedicated to the analysis of the performance of categorical encoding approaches applied to different datasets.

Reply:
the best suited method

Reply:
for your data => scientists data isn't better? I dunnow

Reply:
This can help the data scientist to start encoding smarter. 
I think to remove or refactor. 
It looks like they are stupid now but will be smarter haha :D